README.md

```diff
@@ -1 +1,99 @@
-# hello-world
\ No newline at end of file
+<a name="readme-top"></a>
+📗 Table of Contents
+📖 About the Project
+🛠 Built With
+Tech Stack
+Key Features
+
+💻 Getting Started
+Setup
+👥 Authors
+🔭 Future Features
+⭐️ Show your support
+🙏 Acknowledgements
+❓ FAQ
+📝 License
+
+# 📖 [Hello World] <a name="about-project"></a>
+
+> It is my first Microverse Project, and it has linter to rectify and print out code mistakes.
+
+### HTML and CSS <a name="tech-stack"></a>
+
+> HTML and CSS is a coding langauge that developers can use to build project like this.
+
+###In order to run this project you need:
+
+
+- Github account
+- Git bash
+-node js(installed)
+- well working browser.
+
+### Setup
+
+Clone this repository to your desired folder:
+- git clone git@github.com:(https://github.com/ismailmunyentwari9/hello-world/tree/feature1)
+
```

Comment:
- [OPTIONAL] After cloning, you should have showed the users what next to do by `cd into folder-name`.